audbcards/core/dataset.py

```diff
@@ -455,6 +750,61 @@ def create_datacard_page(dataset: Dataset):
                 fp.write('\n')
 
 
+def create_datasets_page_from_template(datasets: typing.Sequence[Dataset],
+                                       of_basename: str = 'datasets'):
+    r"""Create overview page of datasets.
+
+    Args:
+        datasets: list of datasets
+        of_basename: str
```

Comment:
What is `of_basename`, especially for what stand `of`?

Reply:
Docstring improved in 8364857

Reply:
Thanks, it's easier to understand now.

Do we need to provide `ofbase` at all, or could we also set it to the name of the dataset?

Reply:
If ofpath is not specified it defaults to `None` and uses this path:

```python
        if ofpath is None:
            ofpath = f'datasets/{self._dataset.name}_from_template.rst'
```

I thought that it might be needed to be flexible for testing purposes, but if it is unused, this argument could well be discarded.

Reply:
It might be handy to specify an output path if you want to group datasets into different sub dirs,
but in that case we also need to be able to provide the directory.

So maybe we can leave this argument and go with `'datasets/{self._dataset.name}.rst'` as default.

Reply:
We cannot that easily make the `datasets` folder part of the outfile name as this folder needs also to be known by other functions like `Dataset.player()`. So I would stick with your suggestion that at the moment only the basename can be influenced by it.
For changing the output folder we will need to add an argument to `audbcards.Dataset()`.